docker/image/console/root/lib/task/build/frontend.sh.twig

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+function task_build_frontend()
+{(
+    set -e
+    cd {{ @('frontend.path') }}
+    
+    if [ ! {{ @('frontend.build.when')|raw }} ]; then
+        return 0;
+    fi
+
+    {% for step in @('frontend.build.steps') -%}
+    {{ step|raw }}
+    {% endfor %}
+
+    set +e
```

Comment:
No need to set +e when within a subshell `(...)`

Reply:
updated